CODEOWNERS

```diff
@@ -83,9 +81,7 @@ Makefile* @cilium/build
 /Documentation/check-crd-compat-table.sh @cilium/docs-structure
 /Documentation/check-examples.sh @cilium/docs-structure
 /Documentation/check-helmvalues.sh @cilium/docs-structure
-/Documentation/cilium_operator.rst @cilium/operator
 /Documentation/cmdref/ @cilium/nonexistantteam
-/Documentation/commit-access.rst @cilium/contributing @cilium/docs-structure
```

Comment:
This was renamed in 72e7740245c59832faaf85d24bcc4c10539e3882 to `Documentation/contributing/governance/commit_access.rst`, so probably worth to rename here instead of deleting.